### DISCUSSION TITLE
WebView - Could you add JavaScript injection?

### DISCUSSION THREAD

**Discussion by ImHooya** [nhn/gpm.unity#297] [Ideas]
I am using gpm webview.
I need a function to inject js during development, can you please add it?

**Comment by SangYun-nhn** [comment]
@taehyeon-nhn 
Please check this comment

**Comment by taehyeon-nhn** [comment]
Hello. @ImHooya.

Do you need any other functions besides [ExecuteJavaScript](https://github.com/nhn/gpm.unity/blob/main/docs/WebView/README.en.md#executejavascript)?
Please explain the function in more detail.

Thank you.

**Reply by ImHooya** [reply]
Ummm.
ExcuteJS is good feature but I need [This](https://developer.apple.com/documentation/webkit/wkusercontentcontroller/1537448-adduserscript)
Thank you.

**Reply by taehyeon-nhn** [reply]
Thank you for your reply.
I'll let you know when the function is added.

**Comment by taehyeon-nhn** [comment]
Hello. @ImHooya.

GPM WebView v1.10.0 released.
Please refer to [ReleaseNotes](https://github.com/nhn/gpm.unity/blob/main/docs/WebView/ReleaseNotes.md#1100) and [README document](https://github.com/nhn/gpm.unity/blob/main/docs/WebView/README.en.md#supported-features).

If you have any questions, please feel free to contact us.
Thank you.
